youtube_dl/extractor/bilibili.py

```diff
@@ -82,10 +82,10 @@ def _real_extract(self, url):
                 'formats': formats,
             })
 
-        title = self._html_search_regex('<h1[^>]+title="([^"]+)">', webpage, 'title')
+        title = self._html_search_regex(r'<h1[^>]+title="([^"]+)">', webpage, 'title')
         description = self._html_search_meta('description', webpage)
         timestamp = unified_timestamp(self._html_search_regex(
-            r'<time[^>]+datetime="([^"]+)"', webpage, 'upload time', fatal=False))
+            '<time[^>]+datetime="([^"]+)"', webpage, 'upload time', fatal=False))
```

Comment:
As a rule of thumb, raw strings should be always used for regular expressions. But for now I guess it's enough to only fix ones with invalid escape sequences. At least this is what this PR's title stands for.